src/Console/SessionsCommand.php

```diff
@@ -44,21 +51,59 @@ public function handle(DatabaseManager $db): void
         foreach ($spannerConnectionNames as $name) {
             $connection = $db->connection($name);
             if ($connection instanceof SpannerConnection) {
-                $sessions = $connection->listSessions();
-                $count = count($sessions);
-                $data = [];
-                $this->info("{$connection->getName()} contains {$count} session(s).");
-                foreach ($sessions as $session) {
-                    $data[] = [
-                        $session->getName(),
-                        $session->getCreatedAt(),
-                        $session->getLastUsedAt(),
-                    ];
-                }
-                if (count($data) > 0) {
-                    $this->table($headers, $data);
+                $sessions = $this->makeSessionData($connection);
+                $this->info("{$connection->getName()} contains {$sessions->count()} session(s).");
+                if ($sessions->isNotEmpty()) {
+                    $this->table($headers, $sessions);
                 }
             }
         }
     }
+
+    /**
+     * @param Connection $connection
+     * @return Collection<int, array{ string, string, string }>
+     */
+    protected function makeSessionData(Connection $connection): Collection
+    {
+        $descending = $this->getOrder() === 'desc';
+
+        return $connection->listSessions()
+            ->sortBy(fn(Session $s) => $this->getSortValue($s), descending: $descending)
+            ->map(static fn(Session $s) => [
+                $s->getName(),
+                $s->getCreatedAt()->format('Y-m-d H:i:s'),
+                $s->getLastUsedAt()->format('Y-m-d H:i:s'),
```

Comment:
How about using associative array?
`handle()` function no longer needs to know the `$header` and the order of the results.

```suggestion
                'name' => $s->getName(),
                'createdAt' => $s->getCreatedAt()->format('Y-m-d H:i:s'),
                'lastUsedAt' => $s->getLastUsedAt()->format('Y-m-d H:i:s'),
```

Reply:
sounds good.

Reply:
Implemented.